src/app/[id]/ConnectEmails.tsx

```diff
@@ -74,35 +83,20 @@ const ConnectEmails = () => {
             width={40}
             height={40}
             style={{
-              maxWidth: "100%",
-              height: "auto"
-            }} />
+              maxWidth: '100%',
+              height: 'auto',
+            }}
+          />
           <div className="flex flex-col items-center text-base font-semibold">
             <p className="text-brand-400">
               Click to upload <span className="text-grey-700">or drag and drop</span>
             </p>
             <p className="text-grey-700">(.eml format)</p>
           </div>
-          <Input
-            id="email-file"
-            type="file"
-            accept=".eml"
-            className="hidden"
-            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
so where does this change happen on upload if not here?

Reply:
It is happening in its parent component